core/Cargo.toml

```diff
@@ -39,11 +39,9 @@ libp2p-mplex = { workspace = true }
 libp2p-noise = { workspace = true }
 multihash = { version = "0.17.0", default-features = false, features = ["arb"] }
 quickcheck = { workspace = true }
+libp2p-identity = { workspace = true, features = ["ed25519"] }
```

Comment:
Why was `libp2p-identity` added here as a dev-dependency even though it is already included as a dependency @thomaseizinger?

Reply:
Originally to activate a feature but that probably became unnecessary in an iteration or something :)

Reply:
See https://github.com/libp2p/rust-libp2p/pull/4040#issuecomment-1579246622.

Reply:
Thus do I understand correctly that this line is no longer needed from now on @thomaseizinger?

Reply:
I think the line is needed but I think the feature can be removed!